core/block_svg.js

```diff
@@ -965,6 +977,11 @@ Blockly.BlockSvg.prototype.onMouseMove_ = function(e) {
       }
       this.setDragging_(true);
       this.moveToDragSurface_();
+
+
+      if (!Blockly.draggedConnectionManager_) {
```

Comment:
My plan was for my Gesture code in my other PR to create a DraggingConnectionManager when starting a block drag, and for that same code to manage the life span of the drag manager.  I think what you're suggesting is out of scope for this PR.